index.js

```diff
@@ -1012,7 +1015,8 @@ module.exports = function (log, indexesPath) {
     else return true
   }
 
-  function getMessage(seq, cb) {
+  function getMessage(seq, valueCache, cb) {
+    if (valueCache[seq]) return cb(null, bipf.decode(valueCache[seq], 0))
```

Comment:
What's the performance improvements this cache adds? I thought we could rely on the cache already inside log.get: https://github.com/ssb-ngi-pointer/async-append-only-log/blob/a75f71485afbbda627fb20a6ef8aafd133c3f85f/index.js#L119

Reply:
Without the cache:

```
db query warmup: 1.802s
db query author all: 693.186ms
db query author post msgs: 164.51ms
db query author 10 msgs: 19.311ms
```

With:

```
db query warmup: 611.244ms
db query author all: 495.455ms
db query author post msgs: 153.325ms
db query author 10 msgs: 18.665ms
```

I was actually puzzled why this PR was so much better at startup performance, this seems to somewhat answer that.

This is running with power connected, so it is a bit faster than the numbers I posted earlier. That is why I included both.

Reply:
Ok, I just wanted to be sure we don't have extra caches (they have a memory hit after all). We could also try `Map` cache instead of object cache.